### DISCUSSION TITLE
我的筆記型電腦（筆電）出大問題了

### DISCUSSION THREAD

**Discussion by enoch-lightning** [1101-GS4719-C/Python-Community-GS4719-C#355] [Q&A]
我的筆電剛剛突然變成下這樣，而且一直停在這個畫面已經十分鐘了，有人知道發生什麼事了嗎？我現在不知道該怎麼辦。不知會不會與我沒有把windows 10作業系統（operating system，OS)更新成windows 11 OS有關。
![IMG20220128133136](https://user-images.githubusercontent.com/86505454/151493068-114578ed-bdd4-4da6-8f7c-91706ed49dc3.jpg)
![IMG20220128133144](https://user-images.githubusercontent.com/86505454/151493079-04ba449c-90d0-4f15-9d09-b2ddd9fc1314.jpg)
會變成這樣，是因為要登入電腦的line就變成這樣了。

**Comment by enoch-lightning** [comment]
我剛剛決定一直按住電腦的電源鍵，將其強制關機。
<img width="858" alt="楊光hp筆記型電腦 標註這是電源鍵" src="https://user-images.githubusercontent.com/86505454/151495115-4adf3dce-076a-4330-9681-1b6cf35aaf72.png">
電腦關機後，我在按電源鍵一下將電腦開機，電腦就可以使用了。

**Comment by enoch-lightning** [comment]
我從很久以前就覺得這台電腦怪怪的，打word只要按下沒幾次儲存檔案的快捷鍵(熱鍵)ctrl+s，word的畫面就會被一條線切一半，再按一次ctrl+s會變回正常word視窗，再按一次ctrl+s我的word視窗又會被同樣一條線切一半。而且，當word進入這種按ctrl+s一次會切一半，再按ctrl+s又回到正常的循環，按ctrl+s不能儲存(save)我新打的東西。

**Comment by pekoranoinu** [comment]
謝謝分享電腦的修復方法

**Comment by JunTingLin** [comment]
遇到藍屏除了可以嘗試重開外，
停止代碼是一個很好的東西，以後遇到都以可以把它拿去餵google，像DRIVER_IRQL_NOT_LESS_OR_EQUAL錯誤訊息就是關於NVIDIA顯示卡方面的錯誤，可以上此網上看看[點我](https://answers.microsoft.com/zh-hant/windows/forum/all/%E9%9B%BB%E8%85%A6%E4%B8%8D%E5%AE%9A%E6%99%82/dcf66490-143c-4827-963a-9f2019f470e0)

**Comment by Ben6uo** [comment]
感謝大大的分享 之前我都直接重灌電腦 現在節省了好多時間
